client/dist/index.html

```diff
@@ -1,14 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
+  
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-</head>
-<body>
-  <div id='root'>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Photos Module</title>
+    <link rel="stylesheet" href="./styles.css">
```

Comment:
Remember to change to CSS modules or styled-components. :)

Reply:
Will do. Thanks!